Example/Podfile

```diff
@@ -23,7 +25,8 @@ abstract_target 'Delirium' do
     # The test frameworks
     pod 'Quick',            '~> 0.10'
     pod 'Nimble',           '~> 5.0'
-    pod 'Nimble-Snapshots', '~> 4.2'
+    pod 'Nimble-Snapshots', '~> 4.3'
+    pod 'Nimble-Snapshots/DynamicType'
```

Comment:
- [ ] Add the used pod version number.

This is very important, because this could break the application when this pod gets updated.